### ISSUE TITLE
Create a dev environment using docker-compose 

### ISSUE THREAD

**Issue by qy21gafy** [amosproj/amos2022ws05-shared-desk-mgmt#24]
## User story
1. As a software developer 
2. I want / need a development environment using docker-compose
3. So that I and my teammates have less set up work in total. 

## Acceptance criteria
* Environment fits our architecture design

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by qy21gafy** [comment]
User story, acceptance criteria will be added in the next sprint.
